.github/workflows/build.yml

```diff
@@ -0,0 +1,93 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - README.md
+
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - README.md
```

Comment:
Do we need to ignore changes to docs, managed directories and might be something else?

Reply:
Added the ignore item for docs. Will add more items in a follow-up pull request once this pull request lands.